### ISSUE TITLE
ERR_CONNECTION_RESET

### ISSUE THREAD

**Issue by SoftTools59654** [brimdata/zui#2947]
The following error occurs sometimes

Some of the errors I report. Maybe it is issued in certain cases, but fixing such small errors will improve this very useful and attractive tool

![image](https://github.com/brimdata/zui/assets/150426297/978efd8d-de18-4b27-9c8d-d0e21cc10cbd)

**Comment by philrz** [comment]
@SoftTools59654: Yes, while we may not be able to get to them all immediately, we'd also like to fix these small errors as we become aware of them. However, this particular error you've shown is unfamiliar. Can you provide some detailed steps (and sample data, if necessary) with which you can reliably cause the error to appear?
